native/cocos/editor-support/spine-creator-support/SkeletonRenderer.cpp

```diff
@@ -453,7 +453,8 @@ void SkeletonRenderer::render(float /*deltaTime*/) {
 #if CC_USE_SPINE_3_8
                 attachmentVertices = reinterpret_cast<AttachmentVertices *>(attachment->getRendererObject());
 #else
-                attachmentVertices = reinterpret_cast<AttachmentVertices *>(attachment->getRegion()->rendererObject);
+                auto *tmpMap = static_cast<spine::HashMap<Attachment *, AttachmentVertices *> *>(attachment->getRegion()->rendererObject);
+                attachmentVertices = (*tmpMap)[attachment];
```

Comment:
`(*tmpMap)[attachment]` might insert a new `nullptr` into the `tmpMap`. Is this expected?

Reply:
Under normal circumstances, every attachment should have its corresponding attachmentVertices, so there should be no cases where nullptr exists.

Reply:
If the attachment is always available, consider using `(*tmpMap).at(attachment)`. 
`at` is readonly and will throw exception if not found.

Reply:
spine::HashMap doesn`t contian 'at' operation

Reply:
The implementation of operator[] contians assert
V operator[](const K &key) {
			Entry *entry = find(key);
			if (entry) return entry->_value;
			else {
				assert(false);
				return {};
			}
		}